keyboards/idobo/keymaps/default75/keymap.c

```diff
@@ -35,7 +31,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * '--------------------------------------------------------------------------------------------------------------------------------------'
  */
 
- [_QW] = { /* QWERTY */
+ [0] = { /* QWERTY */
```

Comment:
The `[_QW]`/`[_FN]` notation you had before was fine, actually.

What I meant was that the layers should use a layout macro, rather than the direct switch matrix notation that you have currently.

The layers should be like this:

```c
  [0] = LAYOUT_ortho_5x15( /* QWERTY */
    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_MINS, KC_GRV,  KC_EQL,  KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC, \
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_LBRC, KC_BSLS, KC_RBRC, KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_QUOT, \
    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_HOME, KC_DEL,  KC_PGUP, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_ENT,  \
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_END,  KC_UP,   KC_PGDN, KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, \
    KC_LCTL, KC_LGUI, KC_LALT, MO(_FN), KC_SPC,  KC_SPC,  KC_LEFT, KC_DOWN, KC_RGHT, KC_SPC,  KC_SPC,  MO(_FN), KC_RALT, KC_RGUI, KC_RCTL  \
  ),
```

The layer identifiers can be numeric `[0]` or aliased `[_QW]`, either is fine.

Reply:
Ohhhhh, totally didn't pick up on that. That is just wild. I haven't had a lot of time to go through all the documentation to see what is all available. Is there anything else you see that would make this cleaner?

Reply:
Actually, yours was in pretty good shape. I've seen far worse.